### ISSUE TITLE
About the classification performance of DG-Router.

### ISSUE THREAD

**Issue by Zilong-Zhang003** [shallowdream204/LoRA-IR#2]
I trained the DG-Router on the seven tasks: 'Haze', 'Rain', 'Motion_Blur', 'Low_Light', 'Noise', 'Kernel_Blur', 'JPEG' using the code you provided. After training and convergence, I tested the model and found that the classification accuracy for the 'Noise' task is extremely low, only 15%, while the accuracy for the other tasks is above 99%. Could you please let me know what training dataset you used, and could you provide the multi-task training code that includes the 'Noise' task?
